Renamed "!stats" to "!records"

ridhwandll opened the pull request:
## Renamed the `!stats` command to the `!records` command
- Fixed a typo in the `res\commands.json` file. (`!getreward`'s | **price** -->**prize**)
EDIT:
- The command typed now have to match exactly with the bots command.
**Example: `!dailygiveme` will not work anymore. The user have to type the correct command.**

ridhwandll committed:
Renamed "!stats" to "!records"


Fix the prefix in the roll command's help.

Gaztin commented on res/commands.json:
```diff
@@ -340,7 +340,7 @@
 		"brief": "Gets a random number.",
 		"usage": "roll <min> <max>\nroll <max>",
-		"additional": "**Example:\n** `!roll 4 10`: Gets a random number between `4` and `10` (inclusive).\n `roll 10`: Gets a random number from `0` to `10` (inclusive).",
+		"additional": "**Example:\n** `!roll 4 10`: Gets a random number between `4` and `10` (inclusive).\n `!roll 10`: Gets a random number from `0` to `10` (inclusive).",
```
`!` is not necessarily the prefix being used!

ridhwandll replied:
It is in the commands.json. I have followed the `!top` command 
![image](https://user-images.githubusercontent.com/69952230/100264179-b38ee700-2f78-11eb-8052-a94758acc237.png)

Gaztin replied:
ah crap. all right then

Gaztin commented on res/commands.json:
```diff
@@ -89,9 +89,9 @@
+	"USER_MODULE_RECORDS": {
 		"brief": "Shows statistics about a user's time in the server.",
-		"usage": "stats <Person>",
+		"usage": "records <Person>",
```
Why did you rename it to "records"?

ridhwandll replied:
It is in the TODO, to change that. I makes confusion with the `!status` command. 🙂

Gaztin commented on src/modules/EventModule.cpp:
```diff
@@ -462,7 +462,7 @@ EventModule::EventModule(UmikoBot* client) : Module("event", true)
 		if (configRD.m_TicketIds.size() == 0)
 		{
-			client.createMessage(message.channelId(), "**You have no tickets.**\nYou can buy them by using `!buyticket <amount>`");
+			client.createMessage(message.channelId(), "**You have no tickets.**\nYou can buy them by using `!buytickets <amount>`");
```
Same as earlier. `!` may not be the chosen prefix!

ridhwandll replied:
Okay done 🙂

ridhwandll committed:
The command have to be matched exactly.

For example: !dailyhellothere will not execute the !daily command anymore.
Fix


Another typo fixed


Fix prefix at the ticket command.

ridhwandll commented:
Okay I am closing this. I will open a new PR shortly 🙂
